server/src/banner.rs

```diff
@@ -44,56 +45,104 @@ pub fn warning_line() {
     );
 }
 
+pub enum ParseableVersion {
```

Comment:
@123vivekr Can you move this inside version module below. They are not used anywhere else.

Reply:
Ah yes. Its done